storage/storage.cpp

```diff
@@ -810,7 +813,7 @@ void Storage::OnServerListDownloaded(vector<string> const & urls)
   }
 
   vector<string> const & downloadingUrls =
-      m_downloadingUrlsForTesting.empty() ? urls : m_downloadingUrlsForTesting;
+      m_downloadingUrlsForTesting.empty() ? *m_sessionServerList : m_downloadingUrlsForTesting;
```

Comment:
Is it possible that |m_sessionServerList|==nullptr? Probably it's worth checking it? (CHECK(m_sessionServerList, ("..."))?

Reply:
there is a check at line 789 :)